homeassistant/components/lcn/services.py

```diff
@@ -92,7 +94,9 @@ def __call__(self, call):
         )
 
         address_connection = self.get_address_connection(call)
-        address_connection.dim_output(output.value, brightness, transition)
+        self.hass.async_create_task(
```

Comment:
It's probably better to refactor and register a coroutine function as service handler instead of `__call__`.

Reply:
Great! I didn't know that its possible to pass coroutines to `hass.services.async_register()`.
I also incorporated the refactorings as in #40665.